ui/components/multichain/pages/send/components/domain-input-resolution-cell.tsx

```diff
@@ -0,0 +1,219 @@
+                  style: { width: '12px', height: '12px' },
+                  name: IconName.Snaps,
+                }}
+              />
+            }
+            positionObj={{
+              bottom: '25%',
+              right: '10%',
+              transform: 'scale(1) translate(25%, 70%)',
+              width: '20px',
+              height: '20px',
+            }}
+          >
+            <Identicon address={address} diameter={32} />
+          </BadgeWrapper>
+        </Tooltip>
+        <div className="multichain-send-page__recipient__item__content">
+          <div
+            ref={titleRef}
+            className="multichain-send-page__recipient__item__title"
+            data-testid="multichain-send-page__recipient__item__title"
+          >
+            {isTitleOverflowing ? (
+              <OverflowingTitle />
+            ) : (
+              <Confusable asText input={domainName} />
+            )}
+          </div>
+          <Text color={TextColor.textAlternative}>{ellipsify(address)}</Text>
+          <div className="multichain-send-page__recipient__item__subtitle">
+            <Text
+              color={TextColor.textAlternative}
+              variant={TextVariant.bodySm}
+            >
+              {protocol}
+            </Text>
+          </div>
+        </div>
+      </div>
+    );
+  }
+  ///: END:ONLY_INCLUDE_IF
+  const getTitle = () => {
+    if (domainName && isTitleOverflowing) {
+      return <OverflowingTitle />;
+    } else if (domainName && !isTitleOverflowing) {
+      return <Confusable asText input={domainName} />;
+    }
+    return ellipsify(address);
+  };
+
+  return (
+    <div
+      key={address}
+      className="multichain-send-page__recipient__item"
+      onClick={() => onClick()}
+    >
+      <div className="multichain-send-page__recipient__item__avatar-wrapper">
+        <Identicon address={address} diameter={32} />
+      </div>
+      <div className="multichain-send-page__recipient__item__content">
+        <div
+          ref={titleRef}
+          className="multichain-send-page__recipient__item__title"
+          data-testid="multichain-send-page__recipient__item__title"
+        >
+          {getTitle()}
+        </div>
+        {domainName && (
+          <div className="multichain-send-page__recipient__item__subtitle">
+            <Text color={TextColor.textAlternative}>{ellipsify(address)}</Text>
+          </div>
+        )}
+        {domainType === 'ENS' && (
+          <Text color={TextColor.textAlternative} variant={TextVariant.bodySm}>
+            {protocol}
+          </Text>
+        )}
+      </div>
+    </div>
+  );
+};
+
+DomainInputResolutionCell.propTypes = {
+  domainType: PropTypes.string.isRequired,
+  address: PropTypes.string.isRequired,
+  domainName: PropTypes.string.isRequired,
+  ///: BEGIN:ONLY_INCLUDE_IF(build-flask)
+  resolvingSnap: PropTypes.string.isRequired,
+  ///: END:ONLY_INCLUDE_IF
+  onClick: PropTypes.func,
+  protocol: PropTypes.string,
+};
```

Comment:
Would make it much easier for me to review if this component had a storybook story. Right now I have to install flask, install the snap, rebuild the extension with correct env flags etc... Instead if there was a story I could use the storybook build in the PR

Reply:
Apologies! I just pushed up a stories file